go/engine/identify2_with_uid.go

```diff
@@ -867,6 +871,10 @@ func (e *Identify2WithUID) runIdentifyUI(m libkb.MetaContext) (err error) {
 		_, err = e.state.Result().GetErrorLax()
 	}
 
+	if outcome.IsOK() {
```

Comment:
I don't really understand what's behind this. Why `TrackStatus_NEW_FAIL_PROOFS` isn't considered IsOk. But I tried a bunch of stuff and this seems like the behavior that is desired. The ideal is to find a condition that means that the state of their identity satisfies your tracking of them, or that you don't track them.